src/androidMain/java/org/cru/godtools/tool/internal/parcel/Parcelize.kt

```diff
@@ -0,0 +1,3 @@
+package org.cru.godtools.tool.internal.parcel
+
+actual typealias Parcelize = kotlinx.parcelize.Parcelize
```

Comment:
Could you explain how the multiplatform works for me? I can see we import the package, then for each platform, we also create a similar folder and file(iosMain, jsMain, etc). Is it just exposing the logic to that platform via this ```actual ...```? But why does this java one have to set ```Parcelize``` to  ```kotlinx.parcelize.Parcelize```?

Reply:
touching on everything else I already posted, we wanted to utilize the `kotlinx.parcelize.Parcelize` annotation on the `State` object for Android only. I couldn't directly apply this annotation because it was in a different source set, so I worked around it by providing a `expect`/`actual` annotation and interface that expose the corresponding android functionality when building for android, but are no-ops when built for other platforms.